msteams-platform/tabs/how-to/authentication/tab-sso-overview.md

```diff
@@ -0,0 +1,203 @@
+
+## Enable SSO for a tab app
+
+This section describes the tasks involved in implementing SSO for a tab app. These tasks are language- and framework-agnostic.
+
+To build a tab app that uses Teams SSO to authenticate users:
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; :::image type="content" source="../../../assets/images/authentication/teams-sso-tabs/teams-sso-accesstoken.png" alt-text="Steps to enable SSO for tab" border="false":::
+
+1. Register with Azure AD: Create an Azure AD app to generate an app ID and app ID URI You also configure redirect URI where Azure AD would send the access token in exchange for identity token for the current user logged into Teams.
+2. Configure code: Update the Teams client to handle access token, calling it when a user accesses your app and validating it when received.
+3. Teams manifest: Update your Teams client manifest with the app ID generated on Azure AD and the app ID URI.
+
+## Best practices
+
+Here's a list of best practices:
+
+- **Don’t cache the access token**: Never cache or store the access token in your app's client-side code. Always call `getAuthToken` when you need an access token. Teams Client caches the access token (or request a new one if it expires). This makes sure that there's no accidental leak of your token from your web-app.
+
+## Known limitations
+
+- SSO in teams only work with OAuth 2.0 protocol.  
+- SSO only works with Azure AD. To extend it to different OAuth Identity providers, the flow needs to be implemented.
+- Multiple domains per app is not supported. For this, please read about LOB apps.
+- Tenant Admin Consent: A simple way of consenting on behalf of an organization as a tenant admin is to refer to `https://login.microsoftonline.com/common/adminconsent?client_id=<AAD_App_ID>`.
+- Ask for consent using the Auth API: Another approach for getting Graph scopes is to present a consent dialog using our existing [web-based Azure AD authentication approach](~/tabs/how-to/authentication/auth-tab-aad.md#navigate-to-the-authorization-page-from-your-pop-up-page). This approach involves popping up an Azure AD consent dialog box.
+
+    To ask for additional consent using the Auth API, follow these steps:
+
+    1. The token retrieved using `getAuthToken()` must be exchanged server-side using Azure AD [on-behalf-of flow](/azure/active-directory/develop/v2-oauth2-on-behalf-of-flow) to get access to those other Graph APIs. Ensure you use the v2 Graph endpoint for this exchange.
+    2. If the exchange fails, Azure AD returns an invalid grant exception. There are usually one of two error messages, `invalid_grant` or `interaction_required`.
+    3. When the exchange fails, you must ask for consent. Show some user interface (UI) asking the user to grant other consent. This UI must include a button that triggers an Azure AD consent dialog box using our [Azure AD authentication API](~/concepts/authentication/auth-silent-aad.md).
```

Comment:
In same section we are referring two different auth types. 

This refers to simple auth implementation.
![image](https://user-images.githubusercontent.com/31851992/165985406-72fdfe0b-3460-435f-83f4-75cbd9165aaa.png)

This section refers to silent auth page.
![image](https://user-images.githubusercontent.com/31851992/165985556-1cef7178-0b6e-484b-b9c6-07612db39b08.png)